src/index.js

```diff
@@ -1,27 +1,29 @@
 import "./index.css";
 import itemList from "./itemList";
+import { getScores, postScores } from "./request";
 
-const placeholders = [
-    {
-        name: "Name",
-        score: 20,
-    },
-    {
-        name: "Name",
-        score: 40,
-    },
-    {
-        name: "Name",
-        score: 50,
-    },
-    {
-        name: "Name",
-        score: 10,
-    },
-    {
-        name: "Name",
-        score: 100,
-    },
-];
+const TEST_GAME_ID = "ubc1A0KC5FEKLnzaXFow";
 
-itemList(placeholders);
\ No newline at end of file
+const refreshButton = document.getElementById("refresh-btn");
+const form = document.getElementById("form");
+
+refreshButton.addEventListener("click", () => {
+  getScores(TEST_GAME_ID).then((scores) => {
+    itemList(scores.result);
+  });
+});
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  const body = {
+    user: document.getElementById("name-input").value,
+    score: parseInt(document.getElementById("score-input").value, 10),
+  };
+  postScores(TEST_GAME_ID, body);
+  document.getElementById("name-input").value = "";
+  document.getElementById("score-input").value = "";
+});
+
+getScores(TEST_GAME_ID).then((scores) => {
+  itemList(scores.result);
```

Comment:
- [ ] It is of best practice to destructure your data for easy usage just as I did in the attached image.
![image](https://user-images.githubusercontent.com/86200356/225347405-f5d79022-5dac-4cad-92b5-50d8b14c7c3e.png)